lib/stigtest.pm

```diff
@@ -32,9 +33,11 @@ our $f_report = 'report.html';
 
 # Set default value for 'scap-security-guide' ds file
 our $f_ssg_sle_ds = '/usr/share/xml/scap/ssg/content/ssg-sle12-ds.xml';
```

Comment:
@ShawnHa0 just curious here, will we have ssg-`sle15`-ds.xml?

Reply:
I believe we already have xml for sle15 in oscap-security-guide after installing this package. The specific xml version is determined in the function set_ds_file() below